src/cli_plugin/install/downloaders/http.js

```diff
@@ -1,25 +1,24 @@
-import Wreck from 'wreck';
 import Progress from '../progress';
 import { fromNode as fn } from 'bluebird';
+import Request from 'request';
 import { createWriteStream, unlinkSync } from 'fs';
 
-function sendRequest({ sourceUrl, timeout }) {
-  const maxRedirects = 11; //Because this one goes to 11.
-  return fn(cb => {
-    const req = Wreck.request('GET', sourceUrl, { timeout, redirects: maxRedirects }, (err, resp) => {
-      if (err) {
-        if (err.code === 'ECONNREFUSED') {
-          err = new Error('ENOTFOUND');
-        }
-
-        return cb(err);
-      }
+function sendRequest({ sourceUrl, reqOptions }) {
+  return new Promise((resolve, reject) => {
+    const req = Request.get(sourceUrl, reqOptions);
 
+    req.on('response', (resp) => {
```

Comment:
@ppisljar good to know. But I think it's better this way for coding style consistency, as it's write like that in the rest of code base.